pkg/pillar/zedcloud/send.go

```diff
@@ -148,17 +155,20 @@ func VerifyAllIntf(ctx ZedCloudContext,
 				errorList = append(errorList, err)
 				continue
```

Comment:
Missed setting it to false here.

Reply:
I actually had the change to reset CloudReachable flag here. But, I hadn't committed that because I was testing it along with some other changes. Will fix this.